src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useContext } from 'react';
+import { PeopleContext } from '../../Context';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import { SortValues } from '../../enums/SortValues';
+
+export const PeopleTable: React.FC = () => {
+  const {
+    searchParams,
+    pathname,
+    people,
+    filteredPeople,
+    sortParam,
+    orderParam,
+    sortHandler,
+  } = useContext(PeopleContext);
+
+  // #region functions
+
+  const capitalize = (word: string) =>
+    word.charAt(0).toUpperCase() + word.slice(1);
+
+  const sexColor = (sex: string) => {
+    return classNames({ 'has-text-danger': sex === 'f' });
+  };
+
+  // #endregion
+  // #region getParents
+
+  const getMother = (_motherName: string | null) => {
+    const mother = people.find(_person => _motherName === _person.name);
+
+    if (mother) {
+      const { slug, name } = mother;
+
+      return (
+        <Link
+          className="has-text-danger"
+          to={{ pathname: `/people/${slug}`, search: searchParams.toString() }}
+        >
+          {name}
+        </Link>
+      );
+    }
+
+    return _motherName || '-';
+  };
+
+  const getFather = (_fatherName: string | null) => {
```

Comment:
The parameter `_fatherName` is used but not defined in the `getFather` function. Ensure that `_fatherName` is passed correctly as a parameter.